src/Adapter/MSTest.TestAdapter/ObjectModel/FixtureTestResult.cs

```diff
@@ -1,16 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using UTF = Microsoft.VisualStudio.TestTools.UnitTesting;
+
 namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.ObjectModel;
 
 [Serializable]
-internal sealed class FixtureTestResult
+internal readonly struct FixtureTestResult
```

Comment:
FixtureTestResult is very lightweight that's returned from a method just to read the properties immediately and do nothing more with it. So changed it to struct.